src/main/java/Calculator/Enums/Operator.java

```diff
@@ -0,0 +1,38 @@
+package Calculator.Enums;
+
+import java.util.function.BiFunction;
+
+public enum Operator {
+    PLUS("+", (a, b) -> a + b),
+    MINUS("-", (a, b) -> a - b),
+    MULTIPLE("*", (a, b) -> a * b),
+    DIVIDE("/", (a, b) -> a / b);
+
+    private String symbol;
+    private BiFunction<Integer, Integer, Integer> function;
+
+    Operator(String symbol, BiFunction<Integer, Integer, Integer> function) {
+        this.symbol = symbol;
+        this.function = function;
+    }
+
+    public static int execute(int numBefore, String operator, int numAfter) {
+        int result = numBefore;
+
+        switch (operator) {
+            case "+":
+                result = PLUS.function.apply(numBefore, numAfter);
+                break;
+            case "-":
+                result = MINUS.function.apply(numBefore, numAfter);
+                break;
+            case "*":
+                result = MULTIPLE.function.apply(numBefore, numAfter);
+                break;
+            case "/":
+                result = DIVIDE.function.apply(numBefore, numAfter);
+                break;
+        }
+        return result;
+    }
+}
```

Comment:
```suggestion
public enum Operator {
    PLUS("+", (a, b) -> a.plus(b)),
    ...

    public Number execute(Number numBefore, Number numAfter) {
        return this.function.apply(numBefore, numAfter);
    }     
```
- switch 문을 반드시 쓸 필요가 있을까요 ㅎ; 오히려 이 부분에서 중복이 발생하고 있네요.
- Number 객체를 적극적으로 활용해보는건 어떨까요? 가령, 연산 로직도 Number 객체의 역할로 위임할 수도 있겠네요.

Reply:
java에서 이런 형태는 처음봐서 여기저기 찾아가며 하다보니 중복이 되었군요. 수정할게요.